iphone/Maps/Core/Framework/ProxyObjects/Routing/MWMRoutingManager.mm

```diff
@@ -80,6 +80,7 @@ - (MWMRouteInfo *)routeInfo {
   if (!self.isRoutingActive) { return  nil; }
   routing::FollowingInfo info;
   self.rm.GetRouteFollowingInfo(info);
+  if (!info.IsValid()) { return nil; }
```

Comment:
nit: I don't mind making better formatting at least in this function :)
Like putting on-line expressions on new line without braces here and above/below.